# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/03/31 16:49:54-05:00 nagar@elinux03.watson.ibm.com 
#   ckrm-E5.delays
# 
# init/Kconfig
#   2004/03/31 16:49:53-05:00 nagar@elinux03.watson.ibm.com +3 -4
#   ckrm-E5.delays
# 
# fs/proc/array.c
#   2004/03/31 16:49:53-05:00 nagar@elinux03.watson.ibm.com +3 -4
#   ckrm-E5.delays
# 
diff -Nru a/fs/proc/array.c b/fs/proc/array.c
--- a/fs/proc/array.c	Wed Mar 31 16:53:58 2004
+++ b/fs/proc/array.c	Wed Mar 31 16:53:58 2004
@@ -426,11 +426,10 @@
 		       get_delay(task,runs),
 		       get_delay(task,runcpu_total),
 		       get_delay(task,waitcpu_total),
-		       get_delay(task,iowait_total),
 		       get_delay(task,num_iowaits),
-		       get_delay(task,mem_iowait_total),
-		       get_delay(task,num_memwaits)
-		       
+		       get_delay(task,iowait_total),
+		       get_delay(task,num_memwaits),
+		       get_delay(task,mem_iowait_total)
 		);
 	return res;
 }
diff -Nru a/init/Kconfig b/init/Kconfig
--- a/init/Kconfig	Wed Mar 31 16:53:58 2004
+++ b/init/Kconfig	Wed Mar 31 16:53:58 2004
@@ -261,10 +261,9 @@
 	bool "Enable delay accounting (EXPERIMENTAL)"
 	help
 	  In addition to counting frequency the total delay in ns is also
-	  recorded. CPU delays are specified as cpu-wait and cpu-run. Memory
-	  delay is recorded for minor and major faults. Information is
-	  accessible through /proc/<pid>/delay.
-
+	  recorded. CPU delays are specified as cpu-wait and cpu-run. 
+	  I/O delays are recorded for memory and regular I/O.
+	  Information is accessible through /proc/<pid>/delay.
 
 config KALLSYMS
 	 bool "Load all symbols for debugging/kksymoops" if EMBEDDED
